### DISCUSSION TITLE
Missing sslmode property in postgres config

### DISCUSSION THREAD

**Discussion by iammrsea** [adonisjs/core#1345] [Help] [answered]
I needed to add sslmode property in my postgres config object, checked the declaration file but it appears to be missing. How do I fix this challenge?

**Comment by McSneaky** [comment]
Had something similar with v4 and MariaDB, so it might work or might not work
But for that just adding required config was good enough, since it got passed directly to underlying Knex. With v5 if it works then TS compiler might be lil bit unhappy. Not sure

**Comment by thetutlage** [comment]
Yeah seems like something needs to be fixed

**Comment by iammrsea** [comment, marked as answer]
Actually, I was looking for a way to be able to locally connect to heroku postgres and run ace commands on the database. That was what led me to the sslmode property. The fix is to add **?ssl=no-verify** to the end of the db connection string, and everything now works fine
